ddtrace/debugging/_encoding.py

```diff
@@ -289,17 +290,82 @@ def _get_fields(obj):
         return {s: _safe_getattr(obj, s) for s in get_slots(obj)}
 
 
-def _captured_value_v2(value, level=1):
-    # type: (Any, int) -> Dict[str, Any]
+def _captured_value_v2(value, level=MAXLEVEL, maxlen=MAXLEN, maxsize=MAXSIZE, maxfields=MAXFIELDS):
+    # type: (Any, int, int, int, int) -> Dict[str, Any]
     _type = type(value)
+
+    if _type in BUILTIN_SIMPLE_TYPES:
+        value_repr = repr(value)
+        return (
+            {
+                "type": _qualname(_type),
+                "value": value_repr,
+            }
+            if len(value_repr) <= maxlen
+            else {
```

Comment:
Nit: isn't it clearer to invert the condition and drop the `else`? I've never seen blank if statements before 🙂

Reply:
Ah I think the formatting is to be blamed here. The `if`  is not empty, but evaluates the stuff that comes before, if the condition is met (`A if COND else B`)

Reply:
Ah, ok - makes sense!